packages/slate/src/models/node.js

```diff
@@ -1722,28 +1743,28 @@ class Node {
   }
 
   /**
-   * Insert a `node` at `index`.
+   * Insert a `node` at `path`.
    *
-   * @param {Number} index
+   * @param {Array} path
    * @param {Node} node
    * @return {Node}
    */
 
-  insertNode(index, node) {
-    const keys = this.getKeysAsArray()
-
-    if (keys.includes(node.key)) {
+  insertNode(path, node) {
```

Comment:
I like this change! Good for them to take a `path` and to make the operation logic simpler. Nice job.